cmd/gen-docs/main.go

```diff
@@ -83,3 +89,41 @@ func fatal(msg interface{}) {
 	fmt.Fprintln(os.Stderr, msg)
 	os.Exit(1)
 }
+
+func genHelpMarkdown(cmd *cobra.Command, dir string) error {
```

Comment:
This is a straightforward approach, but requires some changes to `help_topic.go` to gain access to help topic text.

We already generate a `gh environment` pseudo-command for the help topic. How do you feel that, instead of this approach, we inline and change `GenMarkdownTreeCustom` so that includes help topic command instead of skipping them? Then we don't have to build in special treatment for help topics, and we take more control over docs generation.

Reply:
Yes 100% agree. I already started working on that approach after I re-looked at this PR today.